src/mlpack/methods/quic_svd/quic_svd_impl.hpp

```diff
@@ -64,6 +82,9 @@ inline void QUIC_SVD::ExtractSVD(arma::mat& u,
   arma::vec sigmaBar;
   arma::svd(uBar, sigmaBar, vBar, projectedMatSquared);
 
+  // vBar was tarnsposed of vBar
+  vBar = arma::trans(vBar);
```

Comment:
It is mentioned here that we get transposed of vBar after `svd` [link mentioned in 'quic_svd.hpp`](http://www.cc.gatech.edu/~isbell/papers/isbell-quicsvd-nips-2008.pdf).
![Screenshot from 2023-02-14 02-37-09](https://user-images.githubusercontent.com/108261986/218574923-4fa5a9a6-ee0f-4e3a-af2d-29511c79ac6f.png)

Reply:
I'm extremely surprised this change here doesn't break the tests in `quic_svd_test.cpp`.  I am not sure if the implementation returns exactly what the algorithm says, but we should not make this change: it fundamentally changes how `QUIC_SVD` works and will break user expectations.  As the implementation stands (see the test), the original matrix should be reconstructed as `u * sigma * v.t()`.  Let's not change that---instead if you need a transpose for `CFType`, do it in `QUIC_SVDPolicy`.